<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AutoTrim</class>
 <widget class="QWidget" name="AutoTrim">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>650</width>
    <height>356</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="pb_StartTrim">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>320</y>
     <width>110</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Start Trim</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pb_StopTrim">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>320</y>
     <width>110</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Stop Trim</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="gb_Data">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>10</y>
     <width>280</width>
     <height>280</height>
    </rect>
   </property>
   <property name="title">
    <string>Recorded Data</string>
   </property>
   <widget class="QLineEdit" name="e_delta_yaw">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>80</y>
      <width>100</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="e_avg_uAil">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>180</y>
      <width>100</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="e_delta_h">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>130</y>
      <width>100</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="e_avg_uThrot">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>160</y>
      <width>100</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="e_avg_uRud">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>220</y>
      <width>100</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="e_avg_uElev">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>200</y>
      <width>100</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="e_avg_roll">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>40</y>
      <width>100</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="e_avg_airspeed">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>110</y>
      <width>100</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="e_avg_pitch">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>60</y>
      <width>100</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="e_cur_uThrot">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>160</y>
      <width>100</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="e_cur_uElev">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>200</y>
      <width>100</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="e_cur_yaw">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>80</y>
      <width>100</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="e_cur_airspeed">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>110</y>
      <width>100</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="e_cur_pitch">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>60</y>
      <width>100</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="e_cur_roll">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>40</y>
      <width>100</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="e_cur_uRud">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>220</y>
      <width>100</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="e_cur_uAil">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>180</y>
      <width>100</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="e_cur_h">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>130</y>
      <width>100</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>20</y>
      <width>70</width>
      <height>15</height>
     </rect>
    </property>
    <property name="text">
     <string>Current:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>20</y>
      <width>100</width>
      <height>15</height>
     </rect>
    </property>
    <property name="text">
     <string>Avrg/Accum.</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="e_cur_power">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>250</y>
      <width>100</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="e_avg_power">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>250</y>
      <width>100</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="l_Notes_Text">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>290</y>
     <width>581</width>
     <height>31</height>
    </rect>
   </property>
   <property name="palette">
    <palette>
     <active>
      <colorrole role="WindowText">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>255</red>
         <green>0</green>
         <blue>0</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Text">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>255</red>
         <green>0</green>
         <blue>0</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="ButtonText">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>255</red>
         <green>0</green>
         <blue>0</blue>
        </color>
       </brush>
      </colorrole>
     </active>
     <inactive>
      <colorrole role="WindowText">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>255</red>
         <green>0</green>
         <blue>0</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Text">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>255</red>
         <green>0</green>
         <blue>0</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="ButtonText">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>255</red>
         <green>0</green>
         <blue>0</blue>
        </color>
       </brush>
      </colorrole>
     </inactive>
     <disabled>
      <colorrole role="WindowText">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>120</red>
         <green>120</green>
         <blue>120</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Text">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>120</red>
         <green>120</green>
         <blue>120</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="ButtonText">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>120</red>
         <green>120</green>
         <blue>120</blue>
        </color>
       </brush>
      </colorrole>
     </disabled>
    </palette>
   </property>
   <property name="text">
    <string>Units are SI , Angles are deg; disable q-scaling on AP before trimming!</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pb_Connect">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>320</y>
     <width>110</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Connect</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_19">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>190</y>
     <width>46</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>uAil:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_15">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>140</y>
     <width>111</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>h/delta_h:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_16">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>210</y>
     <width>46</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>uElev:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_11">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>50</y>
     <width>46</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>Roll:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_13">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>90</y>
     <width>121</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>Yaw/delta Yaw:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_17">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>230</y>
     <width>46</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>uRud:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_18">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>170</y>
     <width>111</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>uThrot:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_12">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>70</y>
     <width>46</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>Pitch:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_14">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>120</y>
     <width>111</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>Airspeed:</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="gb_Refs">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>20</y>
     <width>130</width>
     <height>70</height>
    </rect>
   </property>
   <property name="title">
    <string>References</string>
   </property>
   <widget class="QLineEdit" name="e_ref_pitch">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>110</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="e_ref_roll">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>110</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="gb_CurrentTrims">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>180</y>
     <width>130</width>
     <height>90</height>
    </rect>
   </property>
   <property name="title">
    <string>Current Trims</string>
   </property>
   <widget class="QLineEdit" name="e_trim_uRud">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>110</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="e_trim_uElev">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>110</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="e_trim_uAil">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>110</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="gb_timerscounters">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>100</y>
     <width>130</width>
     <height>70</height>
    </rect>
   </property>
   <property name="title">
    <string>timers/counters</string>
   </property>
   <widget class="QLineEdit" name="e_counters">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>110</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="e_timeelapsed">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>110</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="label_20">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>260</y>
     <width>111</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>Power [W]:</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
